README.md

```diff
@@ -145,7 +216,7 @@ After the pipeline completes, you should find a file, `numbers.txt`, in your wor
 
 ### Python Processes
```

Comment:
We should probably make mention of the Python bindings to Vital.  Specifically, a good entry point is the self-documenting Jupyter Notebook located in /vital/bindings/python/Kwiver_Python_Demo.ipynb. Perhaps someone with a better feel for the overall architecture can weigh in on how/where this should go.